cloudrun-malware-scanner/bootstrap.sh

```diff
@@ -131,16 +131,16 @@ Log INFO main "Updating ClamAV config files"
 # see clamd.conf documentation:
 # https://manpages.debian.org/bullseye/clamav-daemon/clamd.conf.5.en.html
 updateClamConfigFile /etc/clamav/clamd.conf << EOF
-# This option allows you to specify the upper limit for data size that will be transfered to remote daemon when scanning a single file.
-StreamMaxLength 521M
+# This option allows you to specify the upper limit for data size that will be transferred to remote daemon when scanning a single file.
+StreamMaxLength 1042M
 
 # Sets the maximum amount of data to be scanned for each input file.
 # Archives and other containers are recursively extracted and scanned up to this value.
-MaxScanSize 512M
+MaxScanSize 1024M
 
 # Files larger than this limit won't be scanned.
 # Affects the input file itself as well as files contained inside it (when the input file is an archive, a document or some other kind of container).
-MaxFileSize 512M
+MaxFileSize 1024M
```

Comment:
Thinking through why all of these are separated.... would these ever be different from each other?  hmmm

Reply:
They are different in their behavior so my guess here is that it just allows finer control of scanning in ClamAV. For example there may be a case where huge files are accepted and the we deem that we just bypass scanning so `MaxFileSize` could be set to less than `MaxScanSize`